### DISCUSSION TITLE
hugo 0.123.7 error

### DISCUSSION THREAD

**Discussion by Gordolin** [hbstack/.github#52] [Q&A] [answered]
Hi, i did an an Update from hugo 0.122.0 to 0.123.7.

With this update i get a bunch of errors:

Error: error building site: render: failed to render pages: render of "home" failed:
"C:\Users\...\AppData\Local\hugo_cache\modules\filecache\modules\pkg\mod\github.com\hugomods\search@v0.13.2\layouts\_default\index.searchindex.json:1:4":
 execute of template failed: template: _default/index.searchindex.json:1:4: executing "_default/index.searchindex.json" at <partialCached "search/index.json" .>:
 error calling partialCached: "C:\Users\...\AppData\Local\hugo_cache\modules\filecache\modules\pkg\mod\github.com\hugomods\search@v0.13.2\layouts\partials\search\index.json.html:28:28":
 execute of template failed: template: partials/search/index.json.html:28:28: executing "partials/search/index.json.html" at <partialCached "search/functions/parse-headings" . .>:
 error calling partialCached: "C:\Users\...\AppData\Local\hugo_cache\modules\filecache\modules\pkg\mod\github.com\hugomods\search@v0.13.2\layouts\partials\search\functions\parse-headings.html:2:20":
 execute of template failed: template: partials/search/functions/parse-headings.html:2:20:
 executing "partials/search/functions/parse-headings.html" at <.Fragments.HeadingsMap>: nil pointer evaluating *tableofcontents.Fragments.HeadingsMap
 
 As you can see i do this on a windows notebook. I udated to the newest modules.
 It seems to be a problem within the search module.

**Comment by razonyang** [comment]
Could you provide a public repo for reproducing?

**Comment by razonyang** [comment, marked as answer]
Please try to upgrade `github.com/hugomods/search` to `v0.13.3`, I'm not sure if it work, since I unable to reproduce the issue with same Hugo version and same version of search module on Windows.

![image](https://github.com/hbstack/.github/assets/17720932/4266ad24-e5da-4f69-8b32-7b79a658bb6c)

If not working, please provide a testing repo for reproducing the issue.

**Reply by Gordolin** [reply]
Hi, that worked - great job

I did an Update to the brand new hugo version 0.123.8
The came another bunch of errors up:

Start building sites …
hugo v0.123.8-5fed9c591b694f314e5939548e11cc3dcb79a79c+extended windows/amd64 BuildDate=2024-03-07T13:14:42Z VendorInfo=gohugoio

ERROR js.Build failed: "C:\Users\...\AppData\Local\hugo_cache\modules\filecache\modules\pkg\mod\github.com\hbstack\hb@v0.12.1\assets\hb\js\index.tmpl.ts:20:18": Could not resolve "hb/modules//js/panel"
ERROR js.Build failed: "C:\Users\...\AppData\Local\hugo_cache\modules\filecache\modules\pkg\mod\github.com\hbstack\hb@v0.12.1\assets\hb\js\index.tmpl.ts:125:19": Could not resolve "hb/modules//js/button"
ERROR JSBUILD: failed to transform "hb/init/index.ts" (text/typescript): "C:\Users\...\AppData\Local\hugo_cache\modules\filecache\modules\pkg\mod\github.com\hbstack\hb@v0.12.1\assets\hb\init\index.tmpl.ts:1:7": Could not resolve "index.ts"
ERROR JSBUILD: failed to transform "hb/js/index.ts" (text/typescript): "C:\Users\...\AppData\Local\hugo_cache\modules\filecache\modules\pkg\mod\github.com\hbstack\hb@v0.12.1\assets\hb\js\index.tmpl.ts:3:9": Could not resolve "index.ts"
ERROR TOCSS: failed to transform "hb/scss/index.scss" (text/x-scss): "C:\Users\...\AppData\Local\hugo_cache\modules\filecache\modules\pkg\mod\github.com\hbstack\hb@v0.12.1\assets\hb\scss\index.tmpl.scss:102:1": File to import not found or unreadable: index.scss.
ERROR render of "section" failed: "C:\Users\...\AppData\Local\hugo_cache\modules\filecache\modules\pkg\mod\github.com\hbstack\hb@v0.12.1\layouts\_default\baseof.html:20:8": execute of template failed: template: archives/list.html:20:8: executing "archives/list.html" at <partial "hugopress/body-end" .>: error calling partial: "C:\Users\...\AppData\Local\hugo_cache\modules\filecache\modules\pkg\mod\github.com\hugomods\hugopress@v0.2.3\layouts\partials\hugopress\body-end.html:1:4": execute of template failed: template: partials/hugopress/body-end.html:1:4: executing "partials/hugopress/body-end.html" at <partial "hugopress/functions/render-hooks" (dict "Name" "body-end" "Page" .)>: error calling partial: "C:\Users\...\AppData\Local\hugo_cache\modules\filecache\modules\pkg\mod\github.com\hugomods\hugopress@v0.2.3\layouts\partials\hugopress\functions\render-hooks.html:18:8": execute of template failed: template: partials/hugopress/functions/render-hooks.html:18:8: executing "partials/hugopress/functions/render-hooks.html" at <partialCached .partial $ctx>: error calling partialCached: "C:\Users\...\AppData\Local\hugo_cache\modules\filecache\modules\pkg\mod\github.com\hbstack\hb@v0.12.1\layouts\partials\hugopress\modules\hb\hooks\body-end.html:1:4": execute of template failed: template: partials/hugopress/modules/hb/hooks/body-end.html:1:4: executing "partials/hugopress/modules/hb/hooks/body-end.html" at <partialCached "hb/assets/js" .Page>: error calling partialCached: "C:\Users\...\AppData\Local\hugo_cache\modules\filecache\modules\pkg\mod\github.com\hbstack\hb@v0.12.1\layouts\partials\hb\assets\js.html:28:25": execute of template failed: template: partials/hb/assets/js.html:28:25: executing "partials/hb/assets/js.html" at <resources.Concat>: error calling Concat: resources in Concat must be of the same Media Type, got "text/javascript" and "text/typescript"
Built in 7019 ms
Error: error building site: TOCSS: failed to transform "hb/scss/index.scss" (text/x-scss): "C:\Users\...\AppData\Local\hugo_cache\modules\filecache\modules\pkg\mod\github.com\hbstack\hb@v0.12.1\assets\hb\scss\index.tmpl.scss:102:1": File to import not found or unreadable: index.scss.

**Reply by razonyang** [reply]
Yes, it's known issue, see hbstack/hb#189, I'd recommend using earlier Hugo version for now.

**Comment by razonyang** [comment]
@Gordolin Hi, I noticed you supported our work via PayPal, I mentioned you on the our [thanks page](https://hbstack.dev/thanks/), thank you very much!
